### PR TITLE
[BugFix] Fix RoPE error in Llama 3.1

### PR DESCRIPTION
**Pull request by WoosukKwon**

### CODE REVIEW

**Review by simon-mo — APPROVED** [review]

**Comment by WoosukKwon** [comment]
I'm fixing this PR. Do not merge right now.

**Review by mgoin — APPROVED** [review]

**Comment by simon-mo** [comment]
Please double check with HF before merge!

**Comment by simon-mo** [comment]
Actually, let's merge this so there's nightly, we will double check before release.

**Pull request merged**
